config/mrs/config.go

```diff
@@ -3,22 +3,31 @@ package mrs
 
 import (
 	"github.com/upbound/upjet/pkg/config"
+
+	"github.com/FrangipaneTeam/provider-flexibleengine/pkg/references"
+	"github.com/FrangipaneTeam/provider-flexibleengine/pkg/tools"
 )
 
 // Configure configures individual resources by adding custom ResourceConfigurators.
 func Configure(p *config.Provider) {
 	// flexibleengine_mrs_cluster_v2
 	// https://registry.terraform.io/providers/FlexibleEngineCloud/flexibleengine/latest/docs/resources/mrs_cluster_v2
 	p.AddResourceConfigurator("flexibleengine_mrs_cluster_v2", func(r *config.Resource) {
-		// node_key_pair is sensitive ?
-		r.TerraformResource.Schema["node_key_pair"].Sensitive = true
+		// public_ip
+		r.References["public_ip"] = references.TypeEIPID().WithCustomExtractor(tools.GenerateExtractor(true, "address")).Get()
+		// node_key_pair
+		r.References["node_key_pair"] = config.Reference{
+			Type: tools.GenerateType("ecs", "KeyPair"),
+		}
 	})
 
 	// flexibleengine_mrs_job_v2
 	// https://registry.terraform.io/providers/FlexibleEngineCloud/flexibleengine/latest/docs/resources/mrs_job_v2
 	p.AddResourceConfigurator("flexibleengine_mrs_job_v2", func(r *config.Resource) {
-
-		r.UseAsync = true
+		// cluster_id
+		r.References["cluster_id"] = config.Reference{
+			Type: tools.GenerateType("mrs", "Cluster"),
```

Comment:
In same resource it's recommended to use local type. 

```suggestion
			Type: "Cluster",
```